seatunnel-connectors-v2/CONNECTOR_ARCHITECTURE.md

```
@@ -0,0 +1,357 @@
+```
+SeaTunnelSource<T, SplitT, StateT>
+    ├── createEnumerator()     // 创建分片枚举器
+    ├── createReader()         // 创建数据读取器
+    ├── restoreEnumerator()    // 从检查点恢复
+    ├── getBoundedness()       // 有界/无界流
+    └── getProducedCatalogTables() // 表元数据
+```
+
+### Sink 端架构
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)